doc/classes/VisualShader.xml

```diff
@@ -146,6 +155,7 @@
 			<argument index="4" name="to_port" type="int">
 			</argument>
 			<description>
+				Returns [code]true[/code] if the specified node and port connection exists.
```

Comment:
```suggestion
				Returns [code]true[/code] if the specified node and port connection exist.
```

Reply:
This one was correct. It would be "exist" if it was "connections". (wouldn't it?)

Reply:
The subject is "the specified node and port connection", so it's two items and thus plural.